web/sites/default/settings.php

```diff
@@ -815,3 +815,13 @@
 elseif (getenv('ELASTICSEARCH_URL')) {
   $config['elasticsearch_connector.cluster.live_cluster']['url'] = getenv('ELASTICSEARCH_URL');
 }
+$databases['default']['default'] = array (
```

Comment:
We have to be careful not to commit  changes to this file as it's also used by production. I can help you sort this out (you can move the changes into settings.local.php)

Reply:
can we gitignore?

Reply:
I can talk you through a better way of doing this, @SamMed1

Reply:
Aah, what Lewis said (apologies, didn't read properly!) — things like DB connections should be in `settings.local.php`. See step 4 of https://github.com/cabinetoffice/national-leadership-centre-drupal#installation-and-configuration

Reply:
ok no probs, I don't actually remember changing anything in this file which is strange, @joesb would this have changed during onboarding yesterday perhaps?